Sources/Controllers/Cells/RouteInfoListItemCell.swift

```diff
@@ -56,16 +60,24 @@ final class RouteInfoListItemCell: UITableViewCell {
         }
     }
     
+    override func layoutSubviews() {
+        super.layoutSubviews()
+        separatorInset = UIEdgeInsets(top: 0, left: leftSeparatorInset + OAUtilities.getLeftMargin(), bottom: 0, right: 0)
+    }
+    
     override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
         super.traitCollectionDidChange(previousTraitCollection)
         if traitCollection.hasDifferentColorAppearance(comparedTo: previousTraitCollection) {
             refreshLeftIcon()
+            bottomImageBorderView.layer.borderColor = UIColor.iconColorDefault.cgColor
         }
     }
     
     private func refreshLeftIcon() {
         guard let leftTurnIconDrawable else { return }
-        leftTurnIconDrawable.clr = (traitCollection.userInterfaceStyle == .dark) ? .white : .black
+        let color = (traitCollection.userInterfaceStyle == .dark) ? UIColor.widgetValue.dark : UIColor.widgetValue.light
```

Comment:
Can you use this code?` let color =  UIColor.widgetValue.dark`

Reply:
I can. But clor will be always white. 
<img width="1577" alt="Screenshot 2024-06-07 at 17 14 48" src="https://github.com/osmandapp/OsmAnd-iOS/assets/35684515/4ee7d953-db3a-4228-b858-e29ba3f0b0c0">